<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventLogWidget</class>
 <widget class="QWidget" name="EventLogWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="eventLogGroupBox">
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 127);</string>
     </property>
     <property name="title">
      <string>Event Logger</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTableView" name="eventLogTableView"/>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Event Filter</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Event Type</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Source</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Message</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QComboBox" name="severityFilterCombo">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="sourceFilterEdit">
             <property name="minimumSize">
              <size>
               <width>130</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="messageFilterEdit">
             <property name="minimumSize">
              <size>
               <width>400</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="clearEventLogBtn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>Clear Event Log</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeLogItemBtn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>Remove Log Item</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exportCSVBtn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>Export to CSV</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
